libr/asm/p/asm_x86_nz.c

```diff
@@ -4913,6 +4910,15 @@ static int parseOperand(RAsm *a, const char *str, Operand *op, bool isrepop) {
 			str = ++p;
 		}
 		op->immediate = getnum (a, str);
+		if (op->immediate == -1) {
+			op->is_good_flag = false;
+			char *num_str = strdup (str);
+			if (num_str) {
+				r_str_trim (num_str);
+				op->is_good_flag = !strcmp (num_str, "0xffffffffffffffff");
```

Comment:
that will fail if the string is 0xFFffFFffFF

Reply:
ok ab09af7ff29158b2e78fa4a5dc2861b094fc4638